cli-helpers/README.md

```diff
@@ -73,6 +101,7 @@ FLAGS
   --lastName=<value>   Last name of user
   --orgId=<value>      ID for organisation that user belongs to
   --poolId=<value>     ID of user pool to add user to
+  --operatorOrgId=<value> Operator organisation ID
```

Comment:
Might be good to add `--operatorOrgId=<value>` to the USAGE command on line 66 too

Reply:
Not sure it is required on that line